### PR TITLE
Add CPUProcesses

### PR DESCRIPTION
**Pull request by jpfairbanks**

### CODE REVIEW

File: src/ComputationalResources.jl [hunk 1]
```diff
@@ -56,13 +56,12 @@ end
...
 CPU1(r::AbstractResource) = CPU1(r.settings)
 
+
 """
     CPUThreads()
     CPUThreads(settings)
-
 Indicate that a computation should be performed using the CPU in multi-threaded mode.
 Optionally pass in an object specifying algorithmic settings.
-
```
Comment (Evizero) [inline src/ComputationalResources.jl]: was this on purpose?

**Comment by timholy** [comment]
LGTM, with the exception of the docstring formatting issue that @Evizero noted.

**Comment by jpfairbanks** [comment]
Fixed formatting.

**Comment by timholy** [comment]
Thanks @jpfairbanks!

**Comment by timholy** [comment]
New tag: https://github.com/JuliaLang/METADATA.jl/pull/11091

**Pull request merged**